Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -933,6 +955,22 @@ void ACesium3DTileset::Tick(float DeltaTime) {
       continue;
     }
 
+    // Apply Actor-defined collision settings to the newly-created component. 
+	UCesiumGltfPrimitiveComponent* PrimitiveComponent = static_cast<UCesiumGltfPrimitiveComponent*>(Gltf->GetChildComponent(0));
+	if (PrimitiveComponent != nullptr)
+	{
+		const UEnum* ChannelEnum = StaticEnum<ECollisionChannel>();
+		if (ChannelEnum)
+		{
+			for (int32 ChannelValue = 0; ChannelValue < ECollisionChannel::ECC_MAX; ChannelValue++)
+			{
+				ECollisionResponse response = BodyInstance.GetCollisionResponse().GetResponse(ECollisionChannel(ChannelValue));
+				PrimitiveComponent->SetCollisionResponseToChannel(ECollisionChannel(ChannelValue), response);
```

Comment:
Is it expected that the collision response channel will change at runtime? Or will it pretty much be configured on the tileset and left alone from then on (other than in the editor perhaps)?

In other words, I'm trying to understand if this needs to be done on Tick.

Reply:
Well, for "standard" primitives components, the property can be changed during runtime. It's interesting for gameplay 
purposes. 

But I also understand that because of the large tileset count of, this could be a bad idea. 
I would say "choose what is best for you". You can decide that for Cesium3DTilesets, it's fixed once at beginplay and can't change. (And in my particular case, I don't need to change it at runtime...). 

What is important is that the property is applied to all newly loaded tiles. And I thought that was the best location to do it for each incoming tile. But you know your code better than me !